src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java

```diff
@@ -16,23 +18,36 @@ public class AuthServiceImpl implements AuthService {
 
     private final UserDetailsManager manager;
     private final PasswordEncoder encoder;
+    private final CustomUserDetailsService customUserDetailsService;
```

Comment:
```suggestion
    private final UserDetailsService customUserDetailsService;
```
инъекцию надо делать всегда по интерфейсу